docs/core_docs/docs/integrations/vectorstores/azure_aisearch.mdx

```diff
@@ -27,6 +27,17 @@ import EnvVars from "@examples/indexes/vector_stores/azure_aisearch/.env.example
 
 <CodeBlock language="text">{EnvVars}</CodeBlock>
 
+## About hybrid search
+
+Hybrid search is a feature that combines the strengths of full text search and vector search to provide the best ranking performance. It's enabled by default in Azure AI Search vector stores, but you can select a different search query type by setting the `search.type` property when creating the vector store.
+
+You can read more about hybrid search and how it may improve your search results in the [official documentation](https://learn.microsoft.com/azure/search/hybrid-search-overview).
+
+In some scenarios like retrieval-augmented generation, you may want to enable **semantic ranking** in addition to hybrid search to improve the relevance of the search results. You can enable semantic ranking by setting the `search.type` property to `AzureAISearchQueryType.SemanticHybrid` when creating the vector store.
```

Comment:
nit: may be worth mentioning 'RAG' next to 'retrieval-augmented generation', as some folks may only be familiar with the acronym.